batcher-bot/liquidity/src/settings.ts

```diff
@@ -0,0 +1,36 @@
+import { liquidity_settings } from "./types";
+
+const path = require("path");
+const fs = require("fs-extra");
```

Comment:
Imo it's not a good idea to mix es6 imports, and commonjs requires. I haven't done nodejs + typescript in a long time so I don't remember exactly, but I'm pretty sure you can do:
```ts
import path from 'path'
import fs from 'fs'
```

If the commonjs require is needed for any reason, feel free to ignore this comment

Reply:
TypeScript should output commonjs for you automatically